### ISSUE TITLE
Question about obtaining correctly scaled point clouds

### ISSUE THREAD

**Issue by oumaima-coder-0410** [facebookresearch/map-anything#107]
Hello, I ran inference on a sequence of images using the images and the camera's intrinsic parameters as input. The point cloud I obtained looks visually good, but when I computed the dimensions of a cube in the point cloud, they did not match the real cube I have.
The real cube measures 79mm × 79mm × 79mm, but from MapAnything I obtained dimensions of approximately 0.44 × 0.45 × 0.47. I am not sure about the units, and the size is almost half of the real cube.
Could you please advise how to obtain point clouds with the correct scale?
Thank you in advance for your help! 

<img width="608" height="545" alt="Image" src="https://github.com/user-attachments/assets/9b93ba70-fbf2-4e8d-a6e4-20cb6c773a0f" />

**Comment by joejep** [comment]
I am facing similar issues. I want to pass both intrinsic and extrinsic of the cameras and see if the pointclouds matches with little errors.

**Comment by Nik-V9** [comment]
Hi, thanks for trying out the codebase!

Unfortunately, due to our limited training data, the metric scale estimation is not perfect for all use cases, where in particular we don't have any object centric data in our training. Metric scale estimation is a purely data prior driven problem and the only way to solve this would be by finetuning MapAnything on more in distribution data.

**Comment by Nik-V9** [comment]
Closing issue, please reopen if further assistance is needed.
